contributors/devel/kubemark-guide.md

```diff
@@ -43,62 +50,84 @@ Common workflow for Kubemark is:
 - monitoring test execution and debugging problems
 - turning down Kubemark cluster
 
-Included in descriptions there will be comments helpful for anyone who'll want to
-port Kubemark to different providers.
+(For now) Included in descriptions there will be comments helpful for anyone who’ll
+want to port Kubemark to different providers.
+(Later) When the refactoring mentioned in the above section finishes, we would replace
+these comments with a clean API that would allow kubemark to run on top of any provider.
 
 ### Starting a Kubemark cluster
 
 To start a Kubemark cluster on GCE you need to create an external kubernetes
```

Comment:
Yes that makes sense. Improved explanation in the `Architecture` section about this.